introduction.md

```diff
@@ -6,41 +6,170 @@ layout: documentation
-Many people find that the simplest way to experiment with openHAB is to get a [Raspberry Pi](https://raspberrypi.org) and install [openHABian]({{base}}/installation/openhabian.html); a "hastle-free openHAB setup".
-While openHABian offers a streamlined and simplified way to get up and running quickly, it is a complete openHAB home automation system capable of automating your entire home.
+## What you need to know before you start
 
-Once you have openHAB up and running, the [Configuration]({{base}}/configuration/index.html) article contains everything you need to know to get your openHAB installation talking to different devices around your home.
-You can use [Sitemaps]({{base}}/configuration/sitemaps.html) to control how the status of these devices are displayed on different openHAB User Interfaces, and you can begin to experiment with [Rules]({{base}}/configuration/rules-dsl.html) in order to get the most out of your installation.
+**When home automation just seems to work, it is always the result of hard work.** 
+Home automation is fascinating and requires a considerable investment of your time. 
+Here are some key considerations especially for new users. To be successful, you will need to:
+
+* **Start slowly and one step at a time.**
+* Be prepared to learn
+* Remain flexible in how you want to achieve your goal
+* Celebrate all the small successes
+
+Remember, openHAB is just a computer program. 
+The computer will only do what *you* tell him to do.
+openHAB can provide many default solutions that are easy to setup. 
+On the flip side, the more you insist that everything should look and work exactly as you want it, the more work you will have to do. 
+openHAB is fully customizable, but doing so will require substantial efforts from you.
+
+After you have read the documentation for openHAB, you will have:
+
+* Identified on which computer you will run openHAB
+* Learned how to install openHAB, as well as all other software that is needed to run openHAB (e.g., JAVA)
+* Learned how your smart devices communicate with openHAB; how to make openHAB give commands to your smart devices; and how you can interact with openHAB
+
+Keep your focus. 
+For almost everything, there is more than one way in openHAB to achieve a goal or perform a task. 
+Initially, this can be frustrating and confusing, but it also gives great flexibility and does not mandate the use of certain devices or tools.
+
+You may need to learn more about computers that you currently do.
+But this may not be a problem as you are reading the documentation of software that will enable you to do home automation yourself.
+
+Lastly, be prepared to start a new hobby: home automation. 
+Basic functions can be achieved in openHAB rather quickly, e.g., switch lights on at a certain time. 
+Others will require much more efforts and thoughts, e.g., how do I determine that someone is home, including guests, but not counting pets? 
+The openHAB forum is a great place to learn and discuss. 
```

Comment:
L50 - L53 are a bit redundant compared to the community part..
Was there a background for mentioning this twice?

Reply:
No real reason, but I remember feeling strong about this issue when I wrote it. deleted